app/javascript/shared/components/__tests__/focusTrap.test.jsx

```diff
@@ -0,0 +1,29 @@
+import { h } from 'preact';
+import { render } from '@testing-library/preact';
```

Comment:
```suggestion
import { render, waitFor } from '@testing-library/preact';
```

(as per below comment)